src/Microsoft.VisualStudio.Services.Agent/VstsAgentWebProxy.cs

```diff
@@ -158,7 +164,7 @@ private void LoadProxySetting()
 
             if (string.IsNullOrEmpty(ProxyAddress))
             {
-                foreach (var envVar in new string[] {"VSTS_HTTP_PROXY", "http_proxy"})
+                foreach (var envVar in new string[] { "VSTS_HTTP_PROXY", "HTTP_PROXY" })
```

Comment:
this needs to stay "http_proxy". Environment variables on unix are case sensitive and the convention for proxy related variables it to be lowercase. Same for "no_proxy" above.

Reply:
Actually, a little more research shows "NO_PROXY" should be fine, but "http_proxy" needs to stay lowercase.

https://unix.stackexchange.com/questions/212894/whats-the-right-format-for-the-http-proxy-environment-variable-caps-or-no-ca

Reply:
I will do it all lower-cas. I also saw a description here
https://www.gnu.org/software/wget/manual/html_node/Proxies.html